src/content/learn/react-developer-tools.md

```diff
@@ -1,33 +1,34 @@
 ---
-title: React Developer Tools
+title: Alat pengembang React
```

Comment:
```suggestion
title: React Developer Tools
```

Best not to translate this since it references the name of the React Developer Tools extension.